tests/tools/validate-expected/index.ts

```diff
@@ -87,7 +87,6 @@ if (!isGolden && !expectedFile) {
 }
 
 const reader = new rpc.StreamMessageReader(process.stdin);
-
```

Comment:
Please revert this change. I like my commits to have a single purpose. If you feel this formatting change is necessary please either submit a follow-up formatting PR or split the formatting changes into a separate commit in this PR.